src/scenes/Reports/AcceleratorReportEditForm.tsx

```diff
@@ -118,6 +121,14 @@ const AcceleratorReportEditForm = ({ report }: AcceleratorReportEditFormProps) =
               </Typography>
             </Box>
           )}
+          <HighlightsBox
+            report={record}
+            projectId={projectId}
+            reportId={reportId}
+            reload={() => {}}
```

Comment:
Are these reloads intentioanaly empty? (here, line 158 and 166)

Reply:
Ya, they are used to reload the report object after saving just the one box in the console side only. On the participant side, the user modifies the whole report before saving, so the reload is not called. A better solution would probably be to make the `reload` prop option. I'll work on that